src/styles/_variables.scss

```diff
@@ -1,15 +1,5 @@
-@import "https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700";
-
-$mainFont: "Montserrat", sans-serif;
-
-$black: #000;
-$white: #fff;
-$gray: #f2f2f2;
-$lightThemeBackground: #fff;
-$lightThemeAccept: #6c488a;
-$lightThemeHeader: #696969;
-
-$darkThemeBackground: #180c30;
+@import '_colors.scss';
+@import '_fonts.scss';
```

Comment:
Why you need this import here?

Reply:
Should I import them in the main.scss: file?

Reply:
fonts file for _fonts.scss
and colors for each file what you need

Reply:
If I delete this imports site will crash